Feature/poc architecture (#216)
mojaloop/documentation: meeting-notes/poc-architecture-notes.md

# Meeting Notes from Proof of Concept Architecture Stream

## Pre-planning Meeting **May 19th** 2020
Atendees:
* Sam
* Miguel
* Pedro

TODOS
* Pedro - Send invites for the sessions

> * Sam - Send invites for the kickoff meeting

Reviewer: ```suggestion
* Sam - Send invites for the kickoff meeting. - Done
```

* Sam - Create slack channel - DONE
* Sam - share google doc and space to put docs/notes - DONE - https://docs.google.com/document/d/1AENW5mpHsZd3GvKazsvOdzHOPvj0vLnB04mGMuB9BIE/edit?invite=CNil3bUJ&ts=5ec3b665

> * Sam - create epic in zenhub

Reviewer: ```suggestion
* Sam - create epic in zenhub - Done
```

Notes
* We foollow kanban approach

> * The PoC is not just about performance, let’s not name it performance, name is "Architecture PoC"

Reviewer: ```suggestion
* The PoC is not just about performance, let’s not name it performance, name is "Architecture PoC", but the focus in PI-10 is on Performance
```

Meetings
* Dedicated work sessions - 2 times a week - Tuesday and Thursday 9:30 am UK to 12
* Standups for the other days - 9:30am UK to 12
* Weds we join the OSS standup 15pm UK
* Sprint reviews at the end of the sprint in the afternoon 13:30 UK - confirm slot with the sponsors, Kim, Miller and Bryan
* Kick-off and PI planning session 20th May at 9am UK - 1.5hr

Resources

> * Miguel and Pedro to try to work 4 days a week until end of may, then 2 days a week for the remaining PI

Reviewer: ```suggestion
* Miguel and Pedro to try to work 3 days a week until end of may, then 2.5 days a week for the remaining PI
* Roman P available full-time
* Joran D G and Don C committed for 4hrs each on a weekly basis
* Amar R will help with QA 2.5 days a week in May and full-time after that.
```
